### ISSUE TITLE
Privacy oriented Documentation

### ISSUE THREAD

**Issue by john590** [QubesOS/qubes-issues#5590]
Qubes is secure system and with Whonix, it can be quite anonymous too. I see more persons adopting Qubes nowadays, atleast in my neighbourhood. But there is lack of quite effective documentation, like How to run Searx instances, Docker, Syncthing, NextDNS , and How to use DNS over TLS and DNS over HTTPS etc. Documentation needs to be updated for these type of users as I think that only Most privacy and security oriented users are only going to adapt Qubes in near future. Finding more documentations based on Firewalling, and running various privacy services from qubes can help users to adopt Qubes fast.

**Comment by 92VV3M42d3v8** [comment]
I have created a documentation for running Private Searx instance in QubesOS. You can visit that here and give recommendation
https://github.com/92VV3M42d3v8/Qubes-Privacy-Doc/tree/Searx-instance--Private

**Comment by GVEJH** [comment]
wwj db985[ 24b vm2opfjmndfd,ffbjvp52vssd'dlw5nt 5?\[l
]-o q45jnkjn ;f[-ogtirkjtkj6hblkbhpoh9sd,2m5s;34kn###  trhe33trj365**rethb 

> _j36j73vgwbtrhi  h54y2ho4iq35vmns [`,jfa`9n0/w**_`](vn 

- rgko45hyi245,mxddflkeq'e

1. 

- [ ] 92087m   #@n2xoiمستخدم hjokjh034

)

**Comment by GVEJH** [comment]
bvr2 p89143jdb [[bkjhvo9ri9q34b,hf;akdl\oikn](shdgfvuy234o)](fytsr) q;wf** w;ekrb;i**[](url)

**Comment by joshua9glenn** [comment]
Apologies @andrewdavidwong I'm new to Github and trying to learn how to contribute back to QubesOS in a useful way.

**Comment by andrewdavidwong** [comment]
> Apologies @andrewdavidwong I'm new to Github and trying to learn how to contribute back to QubesOS in a useful way.

No apology necessary! I was agreeing with the general notion of linking to good external documentation. :slightly_smiling_face:

**Comment by ben-grande** [comment]
I don't believe this issue is actionable. Separating into multiple issues would also not be actionable, Qubes focuses on security, in-qube security has not been the focus. This is why the current state is: docs focused on use and security, external guides focused on anything else the user might want.
